xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/http/ResponseHandlers.java

```diff
@@ -137,7 +137,7 @@ private static boolean isNotJson(PushbackInputStream responseBody) {
 
             return true;
         } catch (IOException e) {
-            throw new UncheckedIOException(e);
+            return true;
```

Comment:
does it make sense to leave only one `return true` at the end of the method?

Reply:
in my opinion it's more readable with explicit return than with empty fall-through catch block